lab_1_keywords_tfidf/main.py

```diff
@@ -86,6 +109,15 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> list[str] | N
         list[str] | None: Token sequence without stop words.
         In case of corrupt input arguments, None is returned.
     """
+    if not check_list(tokens, str, True):
+        return None
+    if not check_list(stop_words, str, True):
+        return None
+    result = []
+    for token in tokens:
```

Comment:
can we make it faster? As we know amount of tokens can be infinite, but amount of stop_words is always limited